python/torch_mlir_e2e_test/test_suite/__init__.py

```diff
@@ -8,7 +8,6 @@
 # to the backend contract.
 COMMON_TORCH_MLIR_LOWERING_XFAILS = {
     "QuantizedMLP_basic",
-    "NormalizeModule_basic",
```

Comment:
Can you also add e2e tests testing the other arguments that `norm.ScalarOpt_dim` takes?